aspnetcore/tutorials/nano-server.md

```diff
@@ -72,23 +76,28 @@ Create a file share on the Nano server so that the published application can be
 <!-- literal_block {"ids": [], "classes": ["code", "ps1"], "xml:space": "preserve"} -->
 
 ````
-mkdir C:\PublishedApps\AspNetCoreSampleForNano
+   mkdir C:\PublishedApps\AspNetCoreSampleForNano
    netsh advfirewall firewall set rule group="File and Printer Sharing" new enable=yes
    net share AspNetCoreSampleForNano=c:\PublishedApps\AspNetCoreSampleForNano /GRANT:EVERYONE`,FULL
    ````
 
-After running the above commands you should be able to access this share by visiting `\\<nanoserver-ip-address>\AspNetCoreSampleForNano` in the host machine's Windows Explorer.
+After running the above commands you should be able to access this share by visiting `\\192.168.1.10\AspNetCoreSampleForNano` in the host machine's Windows Explorer.
 
 ## Open port in the Firewall
 
-Run the following commands in the remote session to open up a port in the firewall to listen for TCP traffic.
+Run the following commands in the remote session to open up a port in the firewall to let IIS listen for TCP traffic on port 80/tcp.
 
 <!-- literal_block {"ids": [], "classes": ["code", "ps1"], "xml:space": "preserve"} -->
 
 ````
-New-NetFirewallRule -Name "AspNet5 IIS" -DisplayName "Allow HTTP on TCP/8000" -Protocol TCP -LocalPort 8000 -Action Allow -Enabled True
+New-NetFirewallRule -Name "AspNetCore IIS" -DisplayName "Allow HTTP on TCP/80" `
+  -Protocol TCP -LocalPort 80 -Action Allow -Enabled True
    ````
 
+Note: This code line is divided by a line breaker escape character that helps reading by wrapping too long lines of code. If you copy-paste it directly
+into your Powershell Remoting prompt it may not work as expected. It works very fine though if paste it in the Powershell ISE and execute the script or selected lines
```

Comment:
--> It works fine, though, if you paste it in the PowerShell ISE and execute the script or selected lines